qa/pom.xml

```diff
@@ -248,6 +260,29 @@
             <artifactId>slf4j-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        
+        <!-- needed by Elasticsearch -->
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>transport</artifactId>
+            <version>${elasticsearch-client-transport.version}</version>
+        </dependency>
+        <dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
+	    	<artifactId>log4j-api</artifactId>
+	    	<version>${log4j-to-slf4j.version}</version>
+	    	<scope>test</scope>
+		</dependency>
+		<dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
+	    	<artifactId>log4j-core</artifactId>
+	    	<version>${log4j-to-slf4j.version}</version>
```

Comment:
Instead of log4j, log4j-over-slf4j can be used. Which diverts all logging to slf4j and thus allows to use any logging framework later on. See: https://www.slf4j.org/legacy.html